tests/src/test/scala/com/nvidia/spark/rapids/ParseDateTimeSuite.scala

```diff
@@ -154,7 +155,8 @@ class ParseDateTimeSuite extends SparkQueryCompareTestSuite with BeforeAndAfterE
     CORRECTED_TIME_PARSER_POLICY) {
     df => {
       df.createOrReplaceTempView("df")
-      df.sqlContext.sql("SELECT c0, to_unix_timestamp(c0, 'yyyy/MM') FROM df")
+      val spark = getActiveSession
+      spark.sql("SELECT c0, to_unix_timestamp(c0, 'yyyy/MM') FROM df")
```

Comment:
We may be able to simplify this test. in this test it seems irrelevant if we read from the actual dataframe or via catalog table df 

This could be just 
```Scala
df.selectExpr(" c0", "to_unix_timestamp(c0, 'yyyy/MM')")
```

Reply:
Thanks Gera! I will update this while working on issue -https://github.com/NVIDIA/spark-rapids/issues/12003. Have made a note of it in that issue.